test/build.actions.test.js

```diff
@@ -579,6 +582,23 @@ test('should not fail if extra package does not have actions', async () => {
     path.normalize('/dist/actions/action-zip.zip'))
 })
 
+test('should skip check for action built before while action filter', async () => {
```

Comment:
better description for the test needed. maybe `should always zip files because the action because skipCheck=true`